src/stack.js

```diff
@@ -1,5 +1,26 @@
 'use strict'
 
 export default class Stack {
-  // your code here
+
+  constructor() {
+    this.stack = []
```

Comment:
I would recommend not using an `Array` as storage for your class. Try to use a `Node` class which contains a single element of your stack. The stack would then use a collection of `Node` instances to build up the stack.